router/forwarder.go

```diff
@@ -38,11 +38,11 @@ func (conn *LocalConnection) ensureForwarders() error {
 	}
 
 	forwarder := NewForwarder(conn, encryptor, udpSender, DefaultPMTU)
-	forwarderDF := NewForwarder(conn, encryptorDF, udpSenderDF, DefaultPMTU)
+	forwarderDF := NewForwarderDF(conn, encryptorDF, udpSenderDF, DefaultPMTU)
 	effectivePMTU := forwarderDF.unverifiedPMTU
 	//NB: only forwarderDF can ever encounter EMSGSIZE errors, and
 	//thus perform PMTU verification
-	forwarder.Start(nil)
+	forwarder.Start()
 	forwarderDF.Start(make(chan int, ChannelSize))
```

Comment:
should just be `forwarderDF.Start()` now, i.e. the chan creation can now happen inside that.